src/lib/sidenav/drawer.ts

```diff
@@ -273,6 +274,16 @@ export class MatDrawer implements AfterContentInit, AfterContentChecked, OnDestr
         this._restoreFocus();
       }
     });
+
+    // Handles the ESCAPE keyboard event
```

Comment:
Can you expand this comment a little bit to explain why we're doing it this way as opposed to just `(keydown)`